tests/testthat/test-infix_spaces_linter.R

```diff
@@ -135,3 +135,20 @@ test_that("infix_spaces_linter can allow >1 spaces optionally", {
     infix_spaces_linter(allow_multiple_spaces = FALSE)
   )
 })
+
+test_that("exception for box::use()", {
+  use_declarations <- c(
+    "box::use(a/b)",
+    "box::use(./a/b)",
+    "box::use(
+      a,
+      a/b,
+      ../a,
+      alias = a/b/c[xyz = abc, ...],
+    )"
+  )
+
+  for (decl in use_declarations) {
+    expect_lint(decl, NULL, infix_spaces_linter())
+  }
```

Comment:
This will produce unhelpful test failure messages if the tests ever start failing.
Better to duplicate `expect_lint()` calls, so the reported line number immediately leads to the failing test case.